auraloss/freq.py

```diff
@@ -181,7 +184,10 @@ def __init__(
                 raise ValueError(
                     f"`sample_rate` must be supplied when `perceptual_weighting = True`."
                 )
-            self.prefilter = FIRFilter(filter_type="aw", fs=sample_rate)
+            if self.prefilter is None:
```

Comment:
Can we adjust the logic here? My thinking is that if someone sets `perceptual_weighting=True` we should apply the default filter. If someone also specifies `prefilter=FIRFilter()` then we should also apply this filter. This would create two separate logic branches here.